pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -411,6 +411,7 @@ func (p *Propagator) processEvent() {
 		// the node's owners list.
 		p.removeDependentFromOwners(existingNode, removed)
 	case event.eventType == deleteEvent:
+		p.gc.absentOwnerCache.Add(accessor.GetUID())
```

Comment:
We may need to revisit this when we do synchronous GC. Does absent owner cache mean the deletion has been *started* or actually *finished*?